checkstyle-sonar-plugin/src/main/java/org/sonar/plugins/checkstyle/CheckstyleRulesDefinition.java

```diff
@@ -60,8 +64,38 @@ static void extractRulesData(NewRepository repository, String xmlRulesFilePath,
         ExternalDescriptionLoader.loadHtmlDescriptions(repository, htmlDescriptionFolder);
         try (InputStream resource = CheckstyleRulesDefinition.class
                 .getResourceAsStream("/org/sonar/l10n/checkstyle.properties")) {
-            PropertyFileLoader.loadNames(repository, resource);
+            loadNames(repository, resource);
         }
         SqaleXmlLoader.load(repository, "/com/sonar/sqale/checkstyle-model.xml");
     }
+
+    @VisibleForTesting
```

Comment:
please remove this annotation.
It is bad practice.
Please make method as private.
All testing of it should be done through public methods, cases that are not possible to cover, we use reflection in tests with clear declaration why it is not possible to be done by public methods.